R/unitConverter.R

```diff
@@ -83,12 +82,13 @@
         sourceUnit          = yUnit,
         molWeight           = molWeight,
         molWeightUnit       = ospUnits$`Molecular weight`$`g/mol`
-      ))) %>%
```

Comment:
@IndrajeetPatil I ran the styler and this was changed => you'll have to rebase the other PR.
Can we add `run the styler` to our CODING STANDARD :)